### PR TITLE
Groups treeview incorporated

### PR DESCRIPTION
**Pull request by suryacgust90**

### CODE REVIEW

File: src/components/ComposePage/GroupsPage/ModalBox/groupsModalBox.tsx [hunk 1]
```diff
@@ -57,6 +58,16 @@ const GroupsModalBox: React.FC<MyModalProps> = ({
   const [attrName,setAttrName] = useState('Attribute Name')
   const [datatype,setDatatype] = useState('VARCHAR')
   const [render,setRender] = useState(false)
+  const fetchProjectSchemaInfo = useSelector(projectSchemaInfoSelector);
+  const [treeArray,setTreeArray] = useState([]);
+
+  useEffect(()=>{
+      if(fetchProjectSchemaInfo && fetchProjectSchemaInfo.length > 0){
+        console.log('Its getting changed')
```
Comment (sourcecoursepoc) [inline src/components/ComposePage/GroupsPage/ModalBox/groupsModalBox.tsx]: Remove unwanted console

File: src/components/ComposePage/MainContent/mainContent.tsx [hunk 2]
```diff
@@ -28,15 +28,15 @@ const MainContent = () => {
...
+    dispatch(fetchProjectSchemaInfoAction(2));
   }, []);
   const handleImport = () => {
     setImportClicked(true);
   };
   //DELETE action
   const handleRemove = (uid: string) => {
     const requestBody = {
-      projectUid: 3,
+      projectUid: 2,
```
Comment (sourcecoursepoc) [inline src/components/ComposePage/MainContent/mainContent.tsx]: Diff between 2 & 3?

**Review by sourcecoursepoc — CHANGES_REQUESTED** [review]

File: src/components/ComposePage/MainContent/mainContent.tsx (hunk 2 above)
Comment (suryacgust90) [reply src/components/ComposePage/MainContent/mainContent.tsx]: Dynamic projectUid fetch incorporated

File: src/components/ComposePage/GroupsPage/ModalBox/groupsModalBox.tsx (hunk 1 above)
Comment (suryacgust90) [reply src/components/ComposePage/GroupsPage/ModalBox/groupsModalBox.tsx]: Removed unwanted console

**Review by sourcecoursepoc — APPROVED** [review]

**Pull request merged**